eos/form-factors/mesonic-processes.hh

```diff
@@ -92,6 +92,18 @@ namespace eos
         static constexpr const char * hqe_prefix = "B_s(*)->D_s(*)";
     };
 
+    struct DToPi {
+        using Transition = PToP;
+        static constexpr const char * label = "D->pi";
+        static constexpr const double m_D = 1.275; //PDG 2016
```

Comment:
This value is very weird, the D mass lies around 1860-1870 MeV^2, right?

Reply:
Furthermore, for the sake of compatibility, can you name this mass `m_B` instead of `m_D`? You will have to propagate the change in your other files.